assets/queries/ansible/azure/container_registry_with_no_locks/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy [ result ] {
+	document := input.document[i]
+  	taskContainerRegistry := getTasks(document)[t]["azure_rm_containerregistry"]
+  	taskLock := getTasks(document)[t2]["azure_rm_lock"]
+  
+    taskContainerRegistry.resource_group != taskLock.resource_group
```

Comment:
If the document has multiple resource groups, will not this always give a different result? because it will always find a different group. For example the container points to lock A and it exists lock A, B and C. This will prove always true, as B and C are different from A. But what we want to check is if A exists, right?